birdhouse/env.local.example

```diff
@@ -226,6 +235,35 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 #                                   }
 #"
 
```

Comment:
@tlvu Let me know if you think all this following part should be in the env.local.example.
It is the way I have to implement it for it to work in the env.local, but I wasn't sure if we wanted to add all the info on the example file (such adding the extra repo to AUTODEPLOY_EXTRA_REPOS).

Reply:
Yes these should be in `env.local.example` but simplified since the loop is out.